pkg/plugins/error.go

```diff
@@ -0,0 +1,21 @@
+package plugins
+
+const (
+	UNSIGNED ErrorCode = iota
+	MODIFIED
+	INVALID
```

Comment:
You shouldn't use all-caps for anything but initialisms. Where are these constants used?

```suggestion
	Unsigned ErrorCode = iota
	Modified
	Invalid
```

Reply:
If that's the case then we have some inconsistencies. Maybe something we can add to the linter?

In this case I think they can be un-exported

Reply:
We would have to enable the typecheck linter, but it uncovers a ton of issues, so would be a huge amount of work.